### PR TITLE
refactor(crux): error statuses for route handlers

### PR DESCRIPTION
**Pull request by AmorfEvo**
Route handlers are annotated with api error responses.

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
LGTM, but please wait for at least one more approval.

File: web/crux/src/app/project/project.http.controller.ts
Comment (robot9706) [inline web/crux/src/app/project/project.http.controller.ts:57]: Could you remove this please? It's unused.

**Review by robot9706 — APPROVED** [review]
Minor comment,

**Pull request merged**
